lib/widgets/pin_code_widget.dart

```diff
@@ -39,7 +39,7 @@ class PinCodeWidgetState extends State<PinCodeWidget> {
       widget.userProfileBloc.userStream.first.then(
         (user) async {
           if (user.securityModel.enrolledBiometrics != "") {
-            await Future.delayed(Duration(milliseconds: 500));
+            await Future.delayed(Duration(milliseconds: 100));
```

Comment:
This interval should be more than the fade transition interval in FadeInRoute otherwise in iOS the page would be seen semi transparent.
This happens because the ui freezes immediately once the biometrics prompt is shown.

Reply:
[Done](https://github.com/breez/breezmobile/pull/231/commits/cb2c537092b947c249de3102aa89562edb1da91e)